### PR TITLE
Fix docs/guides/producer/substitutions

### PR DESCRIPTION
**Pull request by 110y**
Fix some values in `docs/guides/producer/substitutions`

### CODE REVIEW

File: site/content/en/guides/producer/substitutions/_index.md [hunk 1]
```diff
@@ -89,7 +89,7 @@ spec:
 
 ```sh
 # create an image substitution and a setter that populates it
-kpt cfg create-subst hello-world/ image-tag nginx:1.7.9 \
+kpt cfg create-subst hello-world/ image-value nginx:1.7.9 \
```
Comment (pwittrock) [inline site/content/en/guides/producer/substitutions/_index.md:92]: also update the other places `image-tag` appears as the substitution name

**Comment by pwittrock** [comment]
Thanks for the PR!

To clarify is something broken, or is it that you find the name confusing?

File: site/content/en/guides/producer/substitutions/_index.md (hunk 1 above)
Comment (110y) [reply site/content/en/guides/producer/substitutions/_index.md:92]: This is the only place that `image-value` is used in this file.

**Comment by 110y** [comment]
@pwittrock 

Thank you responding.

> To clarify is something broken, or is it that you find the name confusing?

The Former is right:
- `image-tag` substitution is not found in that guide markdown file.
- `image` setter is also not found in the file.

File: site/content/en/guides/producer/substitutions/_index.md (hunk 1 above)
Comment (pwittrock) [reply site/content/en/guides/producer/substitutions/_index.md:92]: Search for the string `image-value`:

line 105, 108, 125

Comment (pwittrock) [reply site/content/en/guides/producer/substitutions/_index.md:92]: Woops, you are correct

**Comment by pwittrock** [comment]
@110y Got it, you are right.

**Comment by pwittrock** [comment]
Thanks for running the docs generation btw!

**Comment by 110y** [comment]
@pwittrock 

Thank you for careful reviewing and merging!

**Pull request merged**